Account_Deletion_Policy.md

```diff
@@ -0,0 +1,64 @@
+
+June 18, 2025
+
+## Introduction
+
+This policy helps clients and employees understand under what conditions we delete a user account.
+
+### Goal Statement
+
+The purpose of this policy is to clarify how we honor account deletion requests in general and specifically as to comply
+with various aspects of GDPR.
+
+### Background Statement 
+
+Our account deletion process is both immediate and permanent. While unintentional or inappropriate deletion of a free 
+account is inconvenient, deletion of a paid account may include monetary and data loss. Despite conspicuous warnings, some
+users have misunderstood the nature of account deletion. To that end, we require paid accounts to make their request via 
+email and confirm that they understand that the service will end immediately and all data, not backed up in another location
+will be lost. Unfortunately, this approach creates a security risk. As email is considered insufficient to establish
+identity we require the user to take an authenticated action to request account deletion.
+
+## Definitions 
+
+### Terms
+
+* The word "we" shall mean Fonticons, Inc., all Fonticons, Inc. employees and any individuals contracting with Fonticons, Inc. to complete work.
+* The word Fonticons will be synonymous with Fonticons, Inc. for the purposes of this policy.
+* Employee shall mean an individual directly employed by Fonticons, and all contractors, consultants, temporary employees, or business partners.
+* Fonticons products/services refers to any and all paid or free products and/or services offered by Fonticons.
+* Client shall mean a person or entity who installs or configures part or all of Fonticons product/service for use on a website or product not owned or otherwise controlled by Fonticons.
+* Fonticons system shall mean any computers, communication systems, platforms, and any other information technology systems used by Fonticons, to provide the Font Awesome product and service.
+* Font Awesome product shall mean the icons, their digital representation, and associated icon functionality present in code.
+* Font Awesome service shall mean the technologies that make the Font Awesome product available to clients, users, and site users.
+
+## Policy 
+
+1. This policy applies to all employees.
+1. This policy applies to all Font Awesome accounts.
+1. All employees are required to follow the procedure below for account deletion requests.
+1. Employees must not use this interaction to attempt to up-sell or otherwise retain users beyond the scope outlined in the procedure.
+
+## Procedures
+
+1. Upon receiving a request, ask the client to: 
+   1. Confirm they understand that account deletion is immediate and irrecoverable.
+   1. Confirm they understand that no refund is granted for deleting an account prior to a paid plan's expiration.
+   1. Login to Font Awesome, navigate to https://fontawesome.com/account/billing, select "Manage Subscription in Stripe", and "Cancel subscription".
```

Comment:
If they've already canceled their subscription, I wonder if there is a secondary verification we can do? Maybe providing the license key, or API key? We don't have to address this now @alexpoiry, it was just a thought I had while reviewing.

Reply:
There is, but it is probably better to just build in MFA which will solve all the problems eventually.